ee/clickhouse/queries/session_recordings/clickhouse_session_recording_list.py

```diff
@@ -186,17 +203,19 @@ def format_event_filters(self) -> EventFiltersSQL:
         aggregate_having_clause = ""
         where_conditions = "AND event IN %(event_names)s"
         # Always include $pageview events so the start_url and end_url can be extracted
-        event_names_to_filter: Set[Union[int, str]] = set(["$pageview"])
+        event_names_to_filter: List[Union[int, str]] = ["$pageview"]
```

Comment:
This change is a bit unrelated, but it cleans up a flaky test because event names would change order in the query snapshots.